Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/IdentityProvidingClients/InternalClientKeys.swift

```diff
@@ -12,4 +12,12 @@ public enum InternalClientKeys {
     public static let internalSTSClientKey = AttributeKey<any IdentityProvidingSTSClient>(
```

Comment:
The attribute keys used to save instances of internal SSO / SSO OIDC clients into `identityProperties` of the auth options returned by auth scheme resolvers generated for each public service.